### PR TITLE
Add package "fira-mono"

### PR DESCRIPTION
**Pull request by rycee**
Turns out that the monospace fonts in the Fira font family is not actually in the "fira" package. So add a new package for the monospace fonts. I did not add this to the old "fira" package since the versioning is different.

### CODE REVIEW

File: pkgs/data/fonts/fira-mono/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{stdenv, fetchurl, unzip}:
```
Comment (jagajaga) [inline pkgs/data/fonts/fira-mono/default.nix]: Add spaces between brackets and contents. like `{ a, b }`

File: pkgs/data/fonts/fira-mono/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+stdenv.mkDerivation rec {
+  name = "fira-mono-3.2";
+
+  src = fetchurl {
+    url = http://www.carrois.com/downloads/fira_mono_3_2/FiraMonoFonts3200.zip;
+    sha256 = "0g3i54q8czf3vylgasj62w2n7l1a2yrbyibjlx1qk3awh7fr1r7p";
+  };
+
+  buildInputs = [unzip];
```
Comment (jagajaga) [inline pkgs/data/fonts/fira-mono/default.nix]: And add brackets here too `[ a ]`

File: pkgs/data/fonts/fira-mono/default.nix [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  phases = ["unpackPhase" "installPhase"];
```
Comment (jagajaga) [inline pkgs/data/fonts/fira-mono/default.nix]: And here :)

File: pkgs/data/fonts/fira-mono/default.nix [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    homepage = http://www.carrois.com/fira-4-1/;
+    description = "Monospace font for Firefox OS";
+    longDescription = ''
+      Fira Mono is a monospace font designed by Erik Spiekermann,
+      Ralph du Carrois, Anja Meiners and Botio Nikoltchev of Carrois
+      Type Design for Mozilla Firefox OS. Available in Regular,
+      Medium, and Bold.
+    '';
+    license = licenses.ofl;
+    maintainers = [maintainers.rycee];
```
Comment (jagajaga) [inline pkgs/data/fonts/fira-mono/default.nix]: And here :)

**Comment by rycee** [comment]
Force pushed update commits.

**Pull request merged**